pyproject.toml

```diff
@@ -58,28 +58,27 @@ progressbar2 = "^3.55.0"
 PyYAML = ">=5.1,<7.0"
 scikit-learn = "^1.0.1"
 scipy = "^1.7.2"
-SQLAlchemy = "<1.4.0,>=1.3.0"
+SQLAlchemy = "^1.4,<1.5"
```

Comment:
Could it be changed to `>=1.3,<2.0`? This would retain compatibility with many more projects already using `muttlib`. Or is the 1.3 -> 1.4 upgrade breaking?

Reply:
`<2.0` and `<1.5` is the same since there aren't any versions in `>=1.5, <2.0`, but I think `<2.0` is better.

Regarding `>=1.3`, I can try and check! 

However I'm not sure how to manage this with MLFlow or other libraries not compatible with 1.3, would muttlib need compatibility for the older version of those libraries too?

Reply:
I just started a poetry project with sqlalchemy 1.3 and this branch to test this. In the last commit I attempted the double compatibility and no issues with poetry or pytest :+1: , just need to test it on sqlalchemy 1.3 since poetry.lock set it to 1.4.